components/Downloads/DownloadReleasesTable.tsx

```diff
@@ -2,55 +2,57 @@ import { FormattedMessage } from 'react-intl';
 import Link from 'next/link';
 import { getNodejsChangelog } from '../../util/getNodeJsChangelog';
 import { getNodeApiLink } from '../../util/getNodeApiLink';
-import type { ExtendedNodeVersionData } from '../../types';
+import { useNodeReleases } from '../../hooks/useNodeReleases';
 import type { FC } from 'react';
 
-type DownloadReleasesTableProps = { releases: ExtendedNodeVersionData[] };
+const DownloadReleasesTable: FC = () => {
+  const { releases } = useNodeReleases();
 
-const DownloadReleasesTable: FC<DownloadReleasesTableProps> = ({
-  releases,
-}) => (
-  <table id="tbVersions" className="download-table full-width">
-    <thead>
-      <tr>
-        <td>Version</td>
-        <td>LTS</td>
-        <td>Date</td>
-        <td>V8</td>
-        <td>npm</td>
-        <td>
-          NODE_MODULE_VERSION<Link href="#ref-1">[1]</Link>
-          <span id="backref-1"></span>
-        </td>
-        <td></td>
-      </tr>
-    </thead>
-    <tbody>
-      {releases.map((release, key) => (
-        <tr key={key}>
-          <td data-label="Version">Node.js {release.nodeNumeric}</td>
-          <td data-label="LTS">{release.ltsName}</td>
-          <td data-label="Date">
-            <time>{release.releaseDate}</time>
-          </td>
-          <td data-label="V8">{release.v8}</td>
-          <td data-label="npm">{release.npm}</td>
-          <td data-label="NODE_MODULE_VERSION">{release.modules}</td>
-          <td className="download-table-last">
-            <a href={`https://nodejs.org/download/release/${release.node}`}>
-              <FormattedMessage id="components.downloadReleasesTable.releases" />
-            </a>
-            <a href={getNodejsChangelog(release.node)}>
-              <FormattedMessage id="components.downloadReleasesTable.changelog" />
-            </a>
-            <a href={getNodeApiLink(release.node)}>
-              <FormattedMessage id="components.downloadReleasesTable.docs" />
-            </a>
+  return (
+    <table id="tbVersions" className="download-table full-width">
+      <thead>
+        <tr>
+          <td>Version</td>
+          <td>LTS</td>
+          <td>Date</td>
+          <td>V8</td>
+          <td>npm</td>
+          <td>
+            NODE_MODULE_VERSION<Link href="#ref-1">[1]</Link>
+            <span id="backref-1"></span>
           </td>
+          <td></td>
```

Comment:
nit: perhaps we could make this more accessible by calling the column "Resources" (or similar)

Reply:
Sounds good :)

Reply:
![Screenshot 2023-06-17 at 2 26 05 PM](https://github.com/nodejs/nodejs.org/assets/128208841/89477b7a-452a-4de0-a366-b7dd079ee973)
The style seems weird to me, I'll revert it for now.
Changing the style seems more troublesome than what's worth, so I'll revert it for now.